src/app/address/controllers/address/prepopulate.js

```diff
@@ -1,22 +1,30 @@
 const BaseController = require("hmpo-form-wizard").Controller;
 
+const { API } = require("../../../../lib/config");
 const {
-  API: {
-    PATHS: { GET_ADDRESSES },
-  },
-} = require("../../../../lib/config");
+  createPersonalDataHeaders,
+} = require("@govuk-one-login/frontend-passthrough-headers");
 
 class AddressPrepopulateController extends BaseController {
   async saveValues(req, res, callback) {
     req.session.prepopulatedPostcode = false;
 
+    const headers = {
+      "session-id": req.session.tokenId,
+      session_id: req.session.tokenId,
+      ...createPersonalDataHeaders(
+        `${API.BASE_URL}${API.PATHS.GET_ADDRESSES}`,
```

Comment:
Don't think we need this here because the get addresses lambda doesn't send audit events

Reply:
@nickhealGDS Agree with removing the personal data headers from this api call? (I just want to be sure you're aware of it).

Reply:
I removed it.

Reply:
Thanks! Shall we just revert the other changes in this file and the test file too to keep the diff tidy?